Chaincase.UI/Pages/ReceivePage.razor

```diff
@@ -1,23 +1,30 @@
 ﻿@page "/receive"
 @using Chaincase.UI.Services
+
+@inject IJSRuntime JS
 @inject UIStateService UiStateService
 @inject NavigationManager NavigationManager
+@inject ReceiveViewModel ReceiveViewModel
+@inherits ReactiveComponentBase<ReceiveViewModel>
 
 <ion-content class="ion-padding">
-    <ion-label>Label</ion-label>
-    <IonInput @bind-Value="Label" placeholder="e.g. Dan for bread" />
-    <p>You must write a memo here for your accounting. It's best to write who you will give this address to.</p>
+    <p>Who are you receiving from?</p>
+    <p>This information is never shared with anyone. It will help keep your future activity private.</p>
```

Comment:
I like it 👍👍

Reply:
this & fee text based on https://www.figma.com/file/jdAqIWCVPRXudqiCYfXOUG/Simplified-coin-control-with-presets?node-id=0%3A1 & https://www.figma.com/community/file/916680391812923706/Bitcoin-Wallet-UI-Kit-(work-in-progress)